unified-scheduler-pool/src/lib.rs

```diff
@@ -1192,6 +1196,53 @@ impl<S: SpawnableScheduler<TH>, TH: TaskHandler> ThreadManager<S, TH> {
         );
     }
 
+    fn max_executing_task_count(mode: SchedulingMode, handler_count: usize) -> Option<usize> {
```

Comment:
This seems to be the only (non-test) spot we determine number of executing threads, afaict.

Can you explain why we want to make `exclusively_initialize_current_thread_for_scheduling` take an `Option` instead of a `u32` directly? as well as why this function returns a usize instead of u32?

Reply:
good question, in short, i want to the internal usage of `u32` at `solana-unified-scheduler-logic` to be hidden. see this commit also: 17d5a0bccb0ce948f8e6838c6cf70fc3a73885d0